pkg/nimble/Makefile.include

```diff
@@ -82,6 +82,9 @@ endif
 ifneq (,$(filter nimble_netif,$(USEMODULE)))
   INCLUDES += -I$(RIOTPKG)/nimble/netif/include
 
+  # when using IP, the host should have a higher prio than the netif thread
+  CFLAGS += -DNIMBLE_HOST_PRIO=1
```

Comment:
Is there a way to make the priority be determined less constant and more dependent on the surrounding threads? E.g.

```suggestion
  CFLAGS += -DNIMBLE_HOST_PRIO="(GNRC_NETIF_PRIO-1)"
```

Reply:
Alternatively

```suggestion
  CFLAGS += -DNIMBLE_HOST_PRIO="(NIMBLE_CONTROLLER_PRIO+1)"
```

I guess